File: AccountingFrontend/src/app/models/Invoice.ts
export class Invoice {
    id:number;
    type: number;
    status: number;
    statusText: string = ""
    typeText: string = "";
    amount: number;
    openAmount: number;
    closedAmount: number;
    contractId: number;

    constructor(id: number, type: number,status: number,amount: number, openAmount: number, closedAmount: number, contractId: number) {
        this.id = id;
        this.type = type;
        this.status = status;
        this.amount = amount;
        this.openAmount = openAmount;
        this.closedAmount = closedAmount;
        this.contractId = contractId;
    }
}

File: InsuranceApp/TransactionService/Models/Invoice.cs
namespace TransactionService.Models
{
    public class Invoice
    {
        public int Id { get; set; }
        public int Amount { get; set; }
        public int OpenAmount { get; set; }
        public int ClosedAmount { get; set; }
        public int Status { get; set; }
        public int Type { get; set; }
        public int? ContractId { get; set; }
    }
}
